src/IceRpc/Server.cs

```diff
@@ -647,9 +647,7 @@ await Task.WhenAll(
 
     /// <summary>Returns true if the <see cref="IConnectorListener.AcceptAsync" /> failure can be retried.</summary>
     private static bool IsRetryableAcceptException(Exception exception) =>
-        // The AcceptAsync call can fail with OperationAborted during shutdown if it is accepting a connection while the
-        // listener is disposed.
-        (exception is IceRpcException rpcException && rpcException.IceRpcError != IceRpcError.OperationAborted) ||
+        (exception is IceRpcException rpcException) ||
```

Comment:
You can simplify - maybe `exception is IceRpcException or AuthenticationException`?